__tests__/database.test.js

```diff
@@ -0,0 +1,42 @@
+const db = require("../db/connection");
+const seed = require("../db/seeds/seed");
+const request = require("supertest");
+const app = require("../app");
+const {
+  categoryData,
+  commentData,
+  reviewData,
+  userData,
+} = require("../db/data/development-data/index");
+
+beforeEach(() => seed({ categoryData, commentData, reviewData, userData }));
+afterAll(() => db.end());
+
+describe("/api/categories", () => {
+  describe("GET", () => {
+    it("GET - status 200: responds with array of category objects", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then((category) => {
+          expect(category.body.length).toBe(7);
+        });
+    });
+    it("GET - status 200: response has correct column properties", () => {
+      return request(app)
+        .get("/api/categories")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.length).toBe(7);
```

Comment:
There are not seven categories in the test-data 

- This is related to the comment above in that you are not checking the correct property on your response. 

- Here you have destructured the body off your response and should be accessing the 'categories' property as that is what you have sent back your category data on. 

This should be body.categories.length - you will have to check your test data to see how many categories we should be getting back.

Reply:
ah, I was checking against the dev data and not test data, will amend now